ivy/functional/frontends/numpy/linalg/solving_equations_and_inverting_matrices.py

```diff
@@ -18,3 +18,8 @@ def inv(a):
 # pinv
 def pinv(a, rtol=1e-15, hermitian=False):
     return ivy.pinv(a, rtol)
+
+
+# tensorinv
+def tensorinv(a, ind=2):
+    return ivy.tensorinv(a, )
```

Comment:
Ivy doesn't have `tensorinv`, we should replicate the behaviour either by using compositional implementation of Ivy function, or if you think this is a function that should be in the Ivy functional API, feel free to create an issue and implement it for all backends.

Reply:
@CatB1t  i tried using compositional implementation by calling  ivy.tensordot() on the result of ivy.tensorinv . What do you think?